index.js

```diff
@@ -63,8 +63,18 @@ function SentryLogger(options) {
     this.captureMessage = this.ravenClient.captureMessage
         .bind(this.ravenClient);
     this.sentryProber = options.sentryProber || null;
+    this.sentryProberAlwaysCallback = !options.sentryProberDetectFailuresBy;
```

Comment:
If the settings used to initialize the sentryProber are not supplied, we still callback immediately without waiting on the result.